AutomatedTesting/CMakeLists.txt

```diff
@@ -8,11 +8,32 @@
 
 if(NOT PROJECT_NAME)
     cmake_minimum_required(VERSION 3.22)
-    include(cmake/EngineFinder.cmake OPTIONAL)
+
+    # Utility function to look for an optional 'engine_finder_cmake_path'setting 
+    function(get_engine_finder_cmake_path project_json_file_path path_value)
+        if(NOT $path_value AND EXISTS "${project_json_file_path}")
```

Comment:
The $path_value variable needs to remove the `$`
CMake needs the name of the variable to check if has not been set or is a false value

```suggestion
        if(NOT path_value AND EXISTS "${project_json_file_path}")
```

Reply:
good catch